src/Microsoft.Dnx.Runtime/DependencyManagement/LibraryManager.cs

```diff
@@ -123,15 +123,15 @@ public IList<DiagnosticMessage> GetAllDiagnostics()
                     string errorCode;
                     if (library.Compatible)
```

Comment:
This code says: If you're Unresolved but Compatible then show the error:
`The dependency x in project y does not support framework f`

That doesn't make sense...